src/main/java/com/github/susom/dbgoodies/etl/Etl.java

```diff
@@ -308,9 +312,38 @@ public void start() throws Exception {
      */
     @CheckReturnValue
     SaveAsBigQuery fetchSize(int nbrRows) {
-      fetchSize = nbrRows;
+      this.fetchSize = nbrRows;
       return this;
     }
+
+    /**
+     * configure number of upload workers
+     */
+    @CheckReturnValue
+    SaveAsBigQuery workerNumber(int workerNum) {
+      this.workerNumber = workerNum;
+      return this;
+    }
+
+    /**
+     * configure batch size of each BigQuery insert
+     */
+    @CheckReturnValue
+    SaveAsBigQuery bigQueryBatchSize(int batchSize) {
```

Comment:
Seems this should be `batchSize(int)` to match the above methods.

Reply:
yes. I will use 'batchSize' instead.